index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="style.css">
+    <link
+        href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;500;600;700;800&family=Roboto:wght@300;400;500&display=swap"
+        rel="stylesheet" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"
+        integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <div class="after-body">
+        <header>
+
+            <nav>
+                <a href="#">My Logo</a>
+                <i class="fa-solid fa-bars"></i>
+            </nav>
+
+        </header>
+        <main id="first-page">
+            <article>
+                <h1>I’m Ahmed <br>Glad to see <br>you!</h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
+                    work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                    contact me.</p>
+                <h1 class="lets-connect">LET'S CONNECT</h1>
+                <ul>
+                    <li><a href="#"><img src="images/twitter.png" alt=""></a></li>
+                    <li><a href="#"><img src="images/Linkedin.png" alt=""></a></li>
+                    <li><a href="#"><img src="images/medium.png" alt=""></a></li>
+                    <li><a href="#"><img src="images/github.png" alt=""></a></li>
+                    <li><a href="#"><img src="images/victory.png" alt=""></a></li>
+                </ul>
```

Comment:
- You have done well using a `ul` tag to group your icons, but this section of your page seems to be overflowing from the page. Please fix this 🙏 

Your project:
<img width="278" alt="evqevwe" src="https://user-images.githubusercontent.com/73607512/219173892-45a345a0-4746-4eee-a41c-4ada48a24389.PNG">